### PR TITLE
[10.0][MIG] product_special_type

### PR DESCRIPTION
**Pull request by rousseldenis**
Dig this module up from https://github.com/OCA/sale-workflow/tree/10.0/product_special_type

### CODE REVIEW

**Comment by rousseldenis** [comment]
@nbessi  @guewen

File: product_special_type/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Let create products as :
+ - Global Discount
+ - Delivery Costs
+ - Advance
```
Comment (legalsylvain) [inline product_special_type/readme/DESCRIPTION.rst:4]: Add Fee

**Review by legalsylvain — APPROVED** [review]
A minor remark. LGTM otherwise. Thanks.

**Review by leemannd — APPROVED** [review]

File: product_special_type/readme/DESCRIPTION.rst (hunk 1 above)
Comment (rousseldenis) [reply product_special_type/readme/DESCRIPTION.rst:4]: Done

**Comment by legalsylvain** [comment]
Thanks !

/ocabot merge

**Pull request merged**